content/02.main-text.md

```diff
@@ -27,7 +27,7 @@ Additionally, discussing manuscripts in the open generates valuable post-publica
 Because incentives are commonly lacking to provide public peer review of existing literature [@doi:10.1629/uksg.245], open collaborative reviews --- where authorship is open to anyone who makes a valid contribution --- could help spur more post-publication peer review.
 However, inviting wide authorship brings many technical and social challenges such as how to fairly distribute credit, coordinate the scientific content, and collaboratively manage extensive reference lists.
 
-To address these challenges, we developed a manuscript writing process using the Markdown language, the GitHub platform, and our new Manubot tool for automating manuscript generation.
+To address these challenges, we developed a manuscript writing process using the Markdown language, the GitHub platform, and our new [Manubot tool](https://github.com/greenelab/manubot) for automating manuscript generation.
```

Comment:
I usually link to manubot-rootstock for this purpose, since it's more user facing than the python package. Let's do a check to make sure we link to both at some point in sensible ways.

Reply:
We have been linking to `manubot-rootstock` as the main Manubot repository elsewhere, but I realized that we didn't link to `manubot` anywhere.  Is there a better place for the `manubot` link?  I'm happy to move it.

Reply:
We don't mention the `manubot` python package anywhere, which seems like a problem. I think at some point, we should mention that much of the Manubot infrastructure is implemented in an open source Python package. It's a tough decision where exactly in the Manubot section though.

Reply:
Added the Manubot Python package in dde450d.